packages/cli/README.md

```diff
@@ -102,7 +102,7 @@ The tool will inkove the installation process for the build tools. Make sure to
 the license agreement before proceeding. This process will install the other required files inside the `directory/decompressed` root directory of the android CLI package.
 
 As a result of the build step, the tool will generate a signed APK (`app-release-signed.apk`)
-that can be uploaded to the Play Store. You will also need to deploy a Digital Asset Links file to
+that can be used for testing the app and a signed AppBundle (`app/build/outputs/bundle/release/app-release.aab`) that can be [uploaded to the Play Store](https://android-developers.googleblog.com/2020/08/recent-android-app-bundle-improvements.html). You will also need to deploy a Digital Asset Links file to
```

Comment:
The app bundle is generated at the root of the project (`app-release-bundle.aab`). Here's what the Bubblewrap output looks like:

```
Building the Android App...
	- Generated Android APK at ./app-release-signed.apk
	- Generated Android App Bundle at ./app-release-bundle.aab
	- Generated Digital Asset Links file at ./assetlinks.json


```

Reply:
Whoops.  Okay.  Will fix now